app/Resources/views/page.html.twig

```diff
@@ -288,11 +288,23 @@
         </script>
 
     {% block javascripts deferred %}
-    {% for javascript in javascripts %}
-        <script async src="{{ asset(javascript) }}"></script>
-    {% endfor %}
+    <script>
+        if (
+          window.localStorage &&
+          document.querySelector &&
+          window.addEventListener &&
+          !!(document.createElement('div')).dataset)
+        {
+            var $body = document.querySelector('body');
+            {% for javascript in javascripts %}
+                const $script = document.createElement('script');
```

Comment:
This will redeclare `$script`? `$script{{loop.index}}` will make it unique.

Reply:
Does anything support the features but not `const`?

Reply:
Good spot, thanks.